docs/tutorials/tutorial_params_options_simulate.ipynb

```diff
@@ -0,0 +1,973 @@
+{
```

Comment:
<p>Not working.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>